OpenSim/Common/Test/TestTableSource.osim

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<OpenSimDocument Version="30500">
+	<TheWorld name="TestTableSource">
+          <components>
+            <TableSource__Vec3_ name="tablesource">
+              <filename>aStaticTrial_2.c3d</filename>
```

Comment:
This tag results from the auto-generated implementation of `getConcreteClassName()`. 

How do you do this? `Object::renameType("TableSource__Vec3__", "TableSourceVec3")`?